contracts/mixnet/src/transactions.rs

```diff
@@ -3,15 +3,63 @@ use crate::error::ContractError;
 use crate::helpers::{calculate_epoch_reward_rate, scale_reward_by_uptime};
 use crate::state::StateParams;
 use crate::storage::{
-    config, config_read, decrement_layer_count, gateway_delegations, gateways, gateways_owners,
-    gateways_owners_read, gateways_read, increase_gateway_bond, increase_gateway_delegated_stakes,
-    increase_mix_delegated_stakes, increase_mixnode_bond, increment_layer_count, mix_delegations,
-    mixnodes, mixnodes_owners, mixnodes_owners_read, mixnodes_read, read_gateway_epoch_reward_rate,
-    read_mixnode_epoch_reward_rate, read_state_params, Layer,
+    config, config_read, decrement_layer_count, gateway_delegations, gateway_delegations_read,
+    gateways, gateways_owners, gateways_owners_read, gateways_read,
+    increase_gateway_delegated_stakes, increase_mix_delegated_stakes, increment_layer_count,
+    mix_delegations, mix_delegations_read, mixnodes, mixnodes_owners, mixnodes_owners_read,
+    mixnodes_read, read_gateway_epoch_reward_rate, read_mixnode_epoch_reward_rate,
+    read_state_params, Layer,
```

Comment:
Why not change it to `storage::*`?

Reply:
Yeah, I guess that would clean it up a bit